src/style.css

```diff
@@ -1,3 +1,82 @@
-body {
+@import 'https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap';
+
+* {
+  --main-color: #00ACDC;
+
+  padding: 0;
   margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Roboto, sans-serif;
+  background: #fff;
+  color: #000;
+  text-transform: uppercase;
+  font-size: 12px;
+  line-height: 14px;
+  font-weight: 500;
+}
+
+a {
```

Comment:
<img width="1044" alt="image" src="https://github.com/mate-academy/layout_moyo-header/assets/86789161/a72bb0dd-6ca7-4411-8231-d9d0f1e1477f">

Reply:
The same in other places